test/e2e/storage/testsuites/fsgroupchangepolicy.go

```diff
@@ -150,6 +155,7 @@ func (s *fsGroupChangePolicyTestSuite) DefineTests(driver storageframework.TestD
 			secondPodFsGroup:                  2000,
 			finalExpectedRootDirFileOwnership: 2000,
 			finalExpectedSubDirFileOwnership:  2000,
+			supportsVolumeMountGroup:          true,
```

Comment:
Did we really wanted this to be true? won't this mean that we will call chgrp on azure?

Reply:
Oh I see that this test does not calls `chgrp`. Can we update test description in the name to be more accurate?

Reply:
I added "via chgrp" to the relevant tests